src/rdkafka.h

```diff
@@ -8724,6 +8806,21 @@ void rd_kafka_DeleteGroups(rd_kafka_t *rk,
                            rd_kafka_queue_t *rkqu);
 
 
+/**
+ * @brief List offset for the specified \p topic_partitions.
```

Comment:
I took it from Java documentation, but there is a typo there:
```suggestion
 * @brief List offsets for the specified \p topic_partitions.
```